File: src/Iniciante/3302.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    int i, entrada;
    int testes;
    scanf("%d", &testes);
    for (i = 0; i < testes; i++)
    {
        scanf("%d", &entrada);
        printf("resposta %d: %d\n", i + 1, entrada);
    }

    return 0;
}

File: src/Iniciante/3302.java
package Iniciante;

import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner input = new Scanner(System.in);
        int testes;
        int entrada;
        testes = input.nextInt();
        for (int i = 0; i < testes; i++) {
            entrada = input.nextInt();
            System.out.println("resposta " + (i + 1) + ": " + entrada);

        }
        input.close();
    }
}

